framework/src/media/audio/audio_manager.c

```diff
@@ -2029,7 +2029,7 @@ audio_manager_result_t change_stream_out_device(int card_id, int device_id)
 }
 
 /* TODO policy should be merged logic of focus manager */
-audio_manager_result_t set_stream_policy(audio_manager_stream_policy_t policy, audio_io_direction_t direct)
+audio_manager_result_t set_stream_policy(stream_policy_t policy, audio_io_direction_t direct)
```

Comment:
I think below all functions should be changed, policy -> info

Reply:
This PR is for adding stream_info_t.
I'll modify it after this PR.